tests/unit/misc_test.c

```diff
@@ -244,13 +244,19 @@ CTEST2(misc, test_size_to_str)
 
         , { KiB + 128                      , "~1.12 KiB"  }
         , { KiB + ((25 * KiB) / 100)       , "~1.25 KiB"  }
-        , { MiB + 128                      , "~1.0 MiB"   }
+        , { MiB + 128                      , "~1.00 MiB"   }
```

Comment:
With fix, decimals like "XX.003" will appear like "XX.00", hence the need to adjust expected output slightly.